aibolit/patterns/method_chaining/method_chaining.py

```diff
@@ -0,0 +1,75 @@
+import javalang
+
+import uuid
+from collections import defaultdict
+
+
+class MethodChainFind:
+
+    def __init__(self):
```

Comment:
@lyriccoder Here we should receive where will we look for the method chaining and store it in a class field